Samples/ProviderTelemetry.tests.json

```diff
@@ -18,13 +20,112 @@
                 "VariantAssignmentReason": "Percentile",
                 "VariantAssignmentPercentage": "30",
                 "DefaultWhenEnabled": "Medium",
-                "AllocationId": "CgBF0x9j_Ip5ccCrdiNO",
-                "ETag": "9no6IpCxWQPlFpamQ9HbwhoDm0E157GN9xUEOtyz-xY",
+                "AllocationId": "DER2rF-ZYog95c4CBZoi",
                 "FeatureFlagReference": "/kv/.appconfig.featureflag/Complete",
                 "FeatureFlagId": "3U5fo_c59GC_8JXZCyKB6O0CiDyuh3KfIqJ0-kAl7Gk",
                 "TargetingId": "Aiden"
             }
         },
         "Description": "A basic feature flag with telemetry."
+    },
+    {
+        "FeatureFlagName": "TelemetryVariantPercentile",
+        "Inputs": {
+            "user": "Sami"
+        },
+        "IsEnabled": {
+            "Result": "true"
+        },
+        "Variant": {
+            "Result": {
+                "someOtherKey": {
+                    "someSubKey": "someSubValue"
+                },
+                "someKey4": [
+                    3,
+                    1,
+                    4,
+                    true
+                ],
+                "someKey": "someValue",
+                "someKey3": 3.14,
+                "someKey2": 3
+            }
+        },
+        "Telemetry": {
+            "event_name": "FeatureEvaluation",
+            "event_properties": {
+                "FeatureName": "TelemetryVariantPercentile",
+                "Enabled": "True",
+                "Version": "1.0.0",
+                "Variant": "True_Override",
+                "VariantAssignmentReason": "Percentile",
+                "VariantAssignmentPercentage": "100",
+                "DefaultWhenEnabled": "True_Override",
+                "AllocationId": "YsdJ4pQpmhYa8KEhRLUn",
+                "FeatureFlagReference": "/kv/.appconfig.featureflag/TelemetryVariantPercentile",
+                "FeatureFlagId": "y3s-SF-wP8w4EseOSP6jsgAMj1NhBZbg5TBrpfFWuas",
+                "TargetingId": "Sami"
+            }
+        },
+        "Description": "A Feature Flag with a single variant with 100 allocation to that variant."
+    },
+    {
+        "FeatureFlagName": "Background.Colors",
+        "Inputs": {
+            "user": "Aiden"
+        },
+        "IsEnabled": {
+            "Result": "true"
+        },
+        "Variant": {
+            "Result": {"b": 130, "g": 0, "r": 75}
```

Comment:
As a result of our allocation id change this is now sorted. This isn't what a customer would expect when printing out there nice rgb values...